.github/actions/trivy_scans/action.yaml

```diff
@@ -0,0 +1,51 @@
+name: IaC and container image scans
+description: "Scan IaC configuration files and the generated Docker image for \
+vulnerabilities."
```

Comment:
It seems to work, but my something is very unhappy with string splitting.
<img width="682" height="99" alt="image" src="https://github.com/user-attachments/assets/59f6897e-e7f2-4bd0-84fd-9c1b65b6c5d9" />

Reply:
I add some spaces which seem to be fine by the linter. I did a full reformatting with one of the linters/formatters (YAML by Red Hat) to both new files in the second fixup. Let me know if this looks better/makes your linter also happy.